src/apps/mesoscale-explorer/ui/states.tsx

```diff
@@ -364,11 +368,95 @@ export class SnapshotControls extends PluginUIComponent<{}> {
     }
 }
 
+export class ExplorerInfo extends PluginUIComponent<{}, { isDisabled: boolean, showHelp: boolean }> {
+    state = {
+        isDisabled: false,
+        showHelp: false
+    };
+
+    componentDidMount() {
+        this.subscribe(this.plugin.state.data.behaviors.isUpdating, v => {
+            this.setState({ isDisabled: v });
+        });
+
+        this.subscribe(this.plugin.state.events.cell.stateUpdated, e => {
+            if (!this.state.isDisabled && MesoscaleState.has(this.plugin) && MesoscaleState.ref(this.plugin) === e.ref) {
+                this.forceUpdate();
+            }
+        });
+    }
+
+    setupDriver = () => {
+        // setup the tour of the interface
+        const driverObj = (this.plugin.customState as MesoscaleExplorerState).driver;
```

Comment:
Tthere should be a check if `driver` is actually available. Generally the app should still work without errors or non-functional UI if `driver` is unavailable.